### PR TITLE
2337 cant access connect with user role connect admin

### PR DESCRIPTION
**Pull request by tenthe**
### Purpose

Fix: #2337.
We had a problem with the matching of routes.
If the module route is defined in the `MODULE_NAME.module.ts`, it could happen that the route could not be resolved correctly. To solve the problem, I had to move the route names to the file `app-routing.module.ts`.
Since this file is generated automatically, the template file `appng5.module.mst` had to be changed.

### Description of the changes
- Moving the routes from `.module.ts` to `app-routing.module.ts`.
- The component `help` and its subcomponents were moved to the module `core-ui`, as otherwise the modules were not loaded correctly

### Remarks

PR introduces (a) breaking change(s): no

PR introduces (a) deprecation(s): no

### CODE REVIEW

File: streampipes-platform-services/src/main/java/org/apache/streampipes/ps/PipelineElementTemplateResource.java [hunk 1]
```diff
@@ -92,32 +95,35 @@ public ResponseEntity<?> getById(@Parameter(description = "The id of the pipelin
...
+             tags = {"Pipeline Element Templates"},
+             responses = {
+                 @ApiResponse(responseCode = "200", description = "Template successfully stored")
+             })
   public ResponseEntity<Void> create(
       @RequestBody(description = "The pipeline element template to be stored",
-          content = @Content(schema = @Schema(implementation = PipelineElementTemplate.class)))
-      @org.springframework.web.bind.annotation.RequestBody PipelineElementTemplate entity) {
+                   content = @Content(schema = @Schema(implementation = PipelineElementTemplate.class)))
+      @org.springframework.web.bind.annotation.RequestBody PipelineElementTemplate entity
```
Comment (bossenti) [inline streampipes-platform-services/src/main/java/org/apache/streampipes/ps/PipelineElementTemplateResource.java:105]: I know it has already been there, but could you place replace this by an import statement? 🙂

File: streampipes-platform-services/src/main/java/org/apache/streampipes/ps/PipelineElementTemplateResource.java [hunk 2]
```diff
@@ -149,44 +156,45 @@ public ResponseEntity<Void> delete(
...
               + "with the labels given in the pipeline element template")
-      @RequestParam("overwriteNames") String overwriteNameAndDescription,
-
+      @RequestParam(value = "overwriteNames", defaultValue = "false", required = false)
+      String overwriteNameAndDescription,
       @RequestBody(description = "The data sink invocation that should be configured with the template contents",
-          content = @Content(schema = @Schema(implementation = DataSinkInvocation.class)))
-      @org.springframework.web.bind.annotation.RequestBody DataSinkInvocation invocation) {
+                   content = @Content(schema = @Schema(implementation = DataSinkInvocation.class)))
+      @org.springframework.web.bind.annotation.RequestBody DataSinkInvocation invocation
```
Comment (bossenti) [inline streampipes-platform-services/src/main/java/org/apache/streampipes/ps/PipelineElementTemplateResource.java:178]: same as above

File: streampipes-platform-services/src/main/java/org/apache/streampipes/ps/PipelineElementTemplateResource.java [hunk 3]
```diff
@@ -196,26 +204,27 @@ public ResponseEntity<DataProcessorInvocation> getPipelineElementForTemplate(
       @RequestParam("overwriteNames") String overwriteNameAndDescription,
 
       @RequestBody(description = "The data processor invocation that should be configured with the template contents",
-          content = @Content(schema = @Schema(implementation = DataProcessorInvocation.class)))
-      @org.springframework.web.bind.annotation.RequestBody DataProcessorInvocation invocation) {
+                   content = @Content(schema = @Schema(implementation = DataProcessorInvocation.class)))
+      @org.springframework.web.bind.annotation.RequestBody DataProcessorInvocation invocation
```
Comment (bossenti) [inline streampipes-platform-services/src/main/java/org/apache/streampipes/ps/PipelineElementTemplateResource.java:208]: same as above

File: streampipes-platform-services/src/main/java/org/apache/streampipes/ps/PipelineElementTemplateResource.java [hunk 4]
```diff
@@ -225,8 +234,9 @@ public ResponseEntity<AdapterDescription> getPipelineElementForTemplate(
       @RequestParam("overwriteNames") String overwriteNameAndDescription,
 
       @RequestBody(description = "The adapter that should be configured with the template contents",
-          content = @Content(schema = @Schema(implementation = AdapterDescription.class)))
-      @org.springframework.web.bind.annotation.RequestBody AdapterDescription adapterDescription) {
+                   content = @Content(schema = @Schema(implementation = AdapterDescription.class)))
+      @org.springframework.web.bind.annotation.RequestBody AdapterDescription adapterDescription
```
Comment (bossenti) [inline streampipes-platform-services/src/main/java/org/apache/streampipes/ps/PipelineElementTemplateResource.java:238]: same as above

File: streampipes-platform-services/src/main/java/org/apache/streampipes/ps/PipelineElementTemplateResource.java (hunk 1 above)
Comment (bossenti) [reply streampipes-platform-services/src/main/java/org/apache/streampipes/ps/PipelineElementTemplateResource.java:105]: Ah okay, didn't see this
Does it make sense to keep both?

Comment (tenthe) [reply streampipes-platform-services/src/main/java/org/apache/streampipes/ps/PipelineElementTemplateResource.java:105]: I assume both are required. One is for spring's `RequestBody` and the other is for swagger.
Perhaps there is a way to harmonize this, but I would suggest doing this in another PR as it is independent of the changes from this PR.

**Review by bossenti — APPROVED** [review]

**Pull request merged**